archunit/src/main/java/com/tngtech/archunit/core/domain/JavaMethod.java

```diff
@@ -125,6 +126,14 @@ public String getDescription() {
         return "Method <" + getFullName() + ">";
     }
 
+    @PublicAPI(usage = ACCESS)
+    public boolean isOverridden() {
+        return Stream.concat(getOwner().getAllRawSuperclasses().stream(), getOwner().getAllRawInterfaces().stream())
```

Comment:
Can we pleae add a small test to cover the feature that overriden methods from interfaces are detected?